commit c1ec83b9d24db2a690b21afcefd7f07fe42242c2 (refs/changes/23/1523/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2017-02-15T16:41:15-08:00 (2 years, 8 months ago)
    
    DAPI-337 dapi's hard-filter-volumes-from mutates vm.internal_metadata['volumesfrom'] such that multiple alloc passes from CNAPI-671 breaks

diff --git a/package.json b/package.json
index d7a94c3..f4efff1 100644
--- a/package.json
+++ b/package.json
@@ -9,7 +9,7 @@
     "assert-plus": "1.0.0",
     "async": "1.5.2",
     "bunyan": "1.8.5",
-    "dapi": "git+https://github.com/joyent/sdc-designation.git#2eee9106262f15b6c9ef04cf68e80a29abe44273",
+    "dapi": "git+https://github.com/joyent/sdc-designation.git#2fbeb8d3ba939ab952c52f389c910a46f44b3094",
     "deep-equal": "1.0.1",
     "dox": "0.4.1",
     "effluent-logger": "git+https://github.com/joshwilsdon/effluent-logger.git#d662f161a07f94045ad2afb45442931511c40e51",
